src/Core/src/Core.csproj

```diff
@@ -1,11 +1,18 @@
-﻿<Project Sdk="MSBuild.Sdk.Extras">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;$(MauiPlatforms)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;$(_MauiDotNetTfm);$(MauiPlatforms)</TargetFrameworks>
```

Comment:
Is there any reason we are still building for `netstandard2.1;netstandard2.0`? I would assume that Maui only needs to build for `net6.0` going forward.

Reply:
This assembly gets loaded by MSBuild (.NET Framework in VS), so that is why we have `netstandard2.0`.

I'm sure why we'd have `netstandard2.1`, though?

Reply:
For reference, WinForms only builds for the "current" .NET TFM:

https://github.com/dotnet/winforms/blob/f52d095a67eb0e9d60539dbd1b09da1e26ef3c29/Directory.Build.props#L9-L11

Same for asp.net:

https://github.com/dotnet/aspnetcore/blob/b89eba6c3cda331ee98063e3c4a04267ec540315/src/DefaultBuilder/src/Microsoft.AspNetCore.csproj#L4
and
https://github.com/dotnet/aspnetcore/blob/9d6f7e3b2b7eb12abc3a656827545a4ceb2ce090/eng/Versions.props#L44

Reply:
> This assembly gets loaded by MSBuild (.NET Framework in VS), so that is why we have netstandard2.0.

Really? Why?

Reply:
MSBuild task?